### DISCUSSION TITLE
Not going to next step

### DISCUSSION THREAD

**Discussion by martcpp** [skills/.github#330] [Communicate using Markdown]
GitHub action is not going to next step after doing the instructed step

**Comment by sinsukehlab** [comment]
You should have pushed a commit to the `start-markdown` branch, not to the `main` branch.
You shouldn't have merged the PR in Step 1.

**Reply by sinsukehlab** [reply]
You can re-create the `start-markdown` branch and open another PR.
Then you can push a commit to the `start-markdown` branch and go to Step 2.

**Reply by vinayadasepalli** [reply]
@sinsukehlab I'm unable to move to the next steps here. Could you please review my file?

https://github.com/vinayadasepalli/skills-communicate-using-markdown/pull/1

**Reply by sinsukehlab** [reply]
Your index.md file is still empty. Add an h1 header using the Markdown syntax `# header` to the file, and you will move to the next step.

**Reply by vinayadasepalli** [reply]
Thank. you @sinsukehlab

**Reply by mjbear** [reply]
> You should have pushed a commit to the `start-markdown` branch, not to the `main` branch. You shouldn't have merged the PR in Step 1.

The key is the Github Actions workflows which [hint at which branch they hook in on](https://github.com/skills/communicate-using-markdown/blob/main/.github/workflows/1-add-headers.yml#L12). Live and learn!

**Comment by martcpp** [comment]
Okay

**Comment by Deule** [comment]
The answers in this thread helped me to resolve my difficulties to come to step 2 and afterwards going through the rest of the course fluently.
Thanks a lot!
